README.md

```diff
@@ -91,6 +91,7 @@ bool wallRight();
 bool wallLeft();
 
 void moveForward();  // can result in "crash"
+void moveForward(int distance); // can result in "crash"
```

Comment:
Might as well use the default argument syntax here, rather than method overloading:

```cpp
void moveForward(int distance = 1); // can result in "crash"
```